scripts/renumber_corpus_files.py

```diff
@@ -22,14 +22,16 @@ def parse_arguments():
                         help='the input directories.')
     parser.add_argument('--output-dir', '-o', required=True,
                         help='the output directory.')
-    size_group = parser.add_mutually_exclusive_group()
+    size_group = parser.add_mutually_exclusive_group(required=True)
     size_group.add_argument('--documents', '-d', type=int,
                             help='the number of documents a file should contain.')
     size_group.add_argument('--keep-sizes', '-k', action='store_true',
                             help='do not merge or split files; i.e. only '
                                  'copies files to the output directory.')
-    parser.add_argument('--zeroes', '-Z', type=int, default=4,
-                        help='the number of zeroes in the output files\' names.')
+    parser.add_argument('--jsonl', '-j', action='store_true',
```

Comment:
I think we agreed to match the format of the output to the input. So we won't need the `-j` argument.

Reply:
Ok.